GUIDE.md

```diff
@@ -283,7 +283,7 @@ const fs = require("fs");
 const path = require("path");
 
 module.exports = {
-  versionUpdated: ({ version, dir, exec }) => {
+  versionUpdated: ({ version, type, dir, exec }) => {
```

Comment:
For consistency, what about using `releaseType` everywhere?

Reply:
indeed, that's make sense.
I'll try to fix.